cmd/modelcmd/apicontext.go

```diff
@@ -62,10 +80,10 @@ func NewAPIContext(ctxt *cmd.Context, opts *AuthOpts) (*APIContext, error) {
 	} else {
 		visitors = append(visitors, httpbakery.WebBrowserVisitor)
 	}
-	webPageVisitor := httpbakery.NewMultiVisitor(visitors...)
 	return &APIContext{
 		Jar:            jar,
-		WebPageVisitor: webPageVisitor,
+		jar:            jar,
```

Comment:
this is the sort of "they're never different until someone doesn't understand the code and sets one without setting the other one".